tests/GLPITestCase.php

```diff
@@ -33,7 +33,7 @@
  * ---------------------------------------------------------------------
  */
 
-use atoum\atoum;
+use atoum;
```

Comment:
I dig a bit inside the atoum autoloader. `atoum` and `atoum\atoum` are default aliases for `atoum\atoum\test` class, see https://github.com/atoum/atoum/blob/fd9a339de252ea774ab755b3fd25dd19be2dbefb/classes/autoloader.php#L41-L48

The following patch is probably the best option to point to a class that is known by any IDE, but will expose to the IDE many method that we are probably not supposed to use in our tests.

```diff
diff --git a/tests/GLPITestCase.php b/tests/GLPITestCase.php
index b273c5b29c..bdf3589a7e 100644
--- a/tests/GLPITestCase.php
+++ b/tests/GLPITestCase.php
@@ -33,14 +33,14 @@
  * ---------------------------------------------------------------------
  */
 
-use atoum\atoum;
+use atoum\atoum\test;
 use Glpi\Tests\Log\TestHandler;
 use Monolog\Level;
 use Psr\Log\LogLevel;
 
 // Main GLPI test case. All tests should extends this class.
 
-class GLPITestCase extends atoum
+class GLPITestCase extends test
 {
     private $int;
     protected $has_failed = false;
```

Reply:
`atoum\atoum\test` doesn't work correctly, I don't have the autocompletion for asserters:

![image](https://github.com/glpi-project/glpi/assets/42734840/fc7a17a6-e634-4887-89e8-24061b3cd67d)

With my original changes, it works fine:

![image](https://github.com/glpi-project/glpi/assets/42734840/b9027f87-3d4b-4395-a692-fbc980f45ba1)

Reply:
This class does not seems to exists, and it is probably your IDE that provides this completion. Anyway, I do not have the autocompletion either when using the `atoum\atoum\test` class, so using only `atoum` will not be worse for me as the current state.

Last thing. Is the import necessary? We are outside a namespace, so I guess that removing it will works.

Reply:
I just realized that the class that was found is actually from a formcreator vendor file:  https://github.com/atoum/stubs

I've added this vendor to GLPI and it should now work for everyone.

Should I move the PR to main or are new *dev* dependencies OK for bf ?

Reply:
It is ok on 10.0/bf branch, as it will only impact devs. We will just wait to merge this after the 10.0.12 release, in case it has unexpected side effects on tests results.